### PR TITLE
resource management page wip

### PR DESCRIPTION
**Pull request by davidleger95**

### CODE REVIEW

File: src/pages/ManageResource.tsx [hunk 1]
```diff
@@ -8,11 +8,12 @@ const ManageResource = ({
...
   }
 }: RouteComponentProps<{ resourceName: string }>) => {
+  console.log({ resourceName });
   return (
     <Wrapper>
       <ResourceHeader name={resourceName} />
-      <div>Provisioning</div>
-      <manifold-resource-details resource-label={resourceName} />
+      <manifold-resource-status resource-name={resourceName} />
+      <manifold-resource-credentials resource-name={resourceName} />
```
Comment (dangodev) [inline src/pages/ManageResource.tsx]: 🤔 I know HTML needs `kebab-case` but in React does it do better with `camelCase`? or does it not matter? Just curious

Comment (davidleger95) [reply src/pages/ManageResource.tsx]: I know this works. not sure if camelCase works.

Comment (dangodev) [reply src/pages/ManageResource.tsx]: According to [Using Web Components in React](https://reactjs.org/docs/web-components.html) looks like you’re right–they should appear as HTML attributes. I may have messed up a few.

File: src/pages/ManageResource.tsx [hunk 2]
```diff
@@ -11,8 +12,11 @@ const ManageResource = ({
...
+      <div>
+        <LinkButton to={`${resourceName}/edit`}>Edit</LinkButton>
+      </div>
```
Comment (davidleger95) [inline src/pages/ManageResource.tsx]: Styling needs to be fixed on this. Will do in next PR

**Pull request merged**